packages/swirl-components/src/components/swirl-tree-view-item/swirl-tree-view-item.css

```diff
@@ -72,6 +72,7 @@
   width: 1.5rem;
   height: 1.5rem;
   margin-right: var(--s-space-4);
+  margin-left: var(--s-space-4);
```

Comment:
Issue: This way padding is missing when the item doesn't have an icon.
<img width="786" alt="Bildschirmfoto 2024-09-17 um 09 29 19" src="https://github.com/user-attachments/assets/b496855d-2a3f-4c3e-9ca7-0ffda1ac7f6e">

Reply:
good finding. fixed.